README.md

```diff
@@ -7,13 +7,175 @@
 Web application for creating and managing automated UI test scenarios and generating
  .feature files with executable test specifications for Cucumber.
  
-## Target users
+ 1. [Synopsis](#synopsis)
+ 2. [Target users](#Target users)
+ 3. [What technologies we used](#What technologies we used)
+ 
+##Synopsis
+This tool used for taking test description in Gherkin language from the front end part and storing it in database.
+Test case consists of steps. Cases form suits. Suits are united into project. Project is model 
+of application being tested. 
+  * Firstly we create project.
+  * Then we should add suit.
+  * Next step is creating test cases for suit.
+  * Finally you need to specify steps that form the test case.
+  
+A single step is described with `Given`, `When` or `Then` Gherkin clauses.
+One should specify what's given initially in the `Given` clause.
+`When` clause describes the condition on which `Then` check is activated. Example:
+
+`Given "I open "<pageName>""`\
+`When "On pagination "<paginationName>" I go to first"`\
+`Then "link "<linkName>" from "<containerName>" match  "<regex>""`
+
+In this three-step example condition of a link on first page matching regex on page `pageName` is tested.
+
+##Target users
 Software Testing Engineers who need 
 * a quick and easy way for automation of the UI testing
 * a platform for storing and organizing large number of test scenarios
 * an automated tests reporting and statistics
 
-## Built With
+##Download application
+For downloading application you can use console and clone repository:
+```text
+git clone https://github.com/TAI-EPAM/jdi-cucumber-test-generator.git
+```
+
+##Run web application
+
+1) After downloading you can open project in IntelliJ IDEA. 
+2) Select "maven projects"->"plugins"->"tomcat7".
+3) Execute "run-war" target. Wait for IDEA to build and deploy .war package. 
+4) Open browser and navigate to "localhost:8080/cucumber/swagger-ui.html#/"
+
+##What technologies we used
+
+### [Spring](https://spring.io/docs)
+* [Spring MVC](https://docs.spring.io/spring/docs/current/spring-framework-reference/web.html)
+Spring Web MVC is the web framework that provides architecture of pattern Model-View-Controller. 
```

Comment:
I think putting here just a link would be just ok. No need to write a detailed description.

Reply:
Shortened